13LessonThirteen/homework/index.html

```diff
@@ -1,10 +1,74 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HW13</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <header>
+    <h1>We're Creatives</h1>
+  </header>
+  <main>
+    <section class="mobile-view">
+      <h2>A fantastic price</h2>
+      <article class="feature-box white-bkgrnd">
+        <h3>Silver</h3>
+        <span class="price">€ 20 <b>/m</b></span>
```

Comment:
Due to some intricacies with character-encoding by browsers and different regions, it's best practice to use something called an html-entity for certain special characters. `&euro;` is the one you want here. 
See this page for more info: http://www.w3schools.com/html/html_entities.asp